app/views/layouts/_navbar.html.haml

```diff
@@ -33,10 +33,11 @@
           %i.fa.fa-eye
           View sites
 
-      .btn-group
-        %button.btn.btn-primary.navbar-btn{ data: { behavior: 'export', toggle: 'tooltip', placement: 'top', container: 'body' }, title: 'Click here to export data for visible sites' }
-          %i.fa.fa-cloud-download
-          Export data
+      - if signed_in?
```

Comment:
Should really have this, instead we either need to make sure that current_user always returns a valid user object by having it return User.new if they are not logged in.